pyproject.toml

```diff
@@ -3,25 +3,22 @@ name = "BioStructBenchmark"
 version = "0.0.1"
 authors = [
     { name="Delta Kapp", email="deltakapp@gmail.com"},
-    { name="Morgan Esler", email="mesler1397@gmail.com"},
+    { name="Morgan Esler", email="mesler1397@gmail.com"}
 ]
 description = "A tool for analyzing protein-nucleic acid complexes"
 readme = "README.md"
-requires-python = ">-3.13"
+requires-python = ">=3.13"
 classifiers = [
     "Programming Language :: Python :: 3",
     "License :: OSI Approved :: MIT License",
     "Operating System :: OS Independent",
 ]
 license = { file = "LICENSE"}
 
+
+[project.urls]
+"Homepage" = "https://github.com/yourusername/hellopkg"
+
 [build-system]
-requires = [
```

Comment:
likely numpy and biopython are needed here

only imports for building wheels is needed here, i.e., you don't need linters